portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,9 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#hello-container{  
+    font-size: 2vh; 
```

Comment:
`vh` for font size was new to me. This seems clever. (No action needed.)

Reply:
Thank you! I find it to be easier to adjust to screen size :)